packages/components/src/apollo/EventsFederationApolloClient.ts

```diff
@@ -159,6 +170,23 @@ class EventsFederationApolloClient {
     return new HttpLink(options);
   }
 
+  /**
+   * Add custom headers to the requests.
+   * NOTE: If using router as a proxy to a subgraph,
+   * remember to propage all these headers to the subgraph
+   * */
+  getCustomHeadersMiddleware() {
+    return new ApolloLink((operation, forward) => {
+      operation.setContext(({ headers = {} }) => ({
+        headers: {
+          ...headers,
+          ...ignoredErrorCodesHeader,
```

Comment:
Should be read from apollo-client config propperty which is populated by appConfig. Chech if there are any to deal the nulls